mongodbatlas/data_source_mongodbatlas_federated_settings_connected_organizations.go

```diff
@@ -0,0 +1,192 @@
+package mongodbatlas
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	matlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+func dataSourceMongoDBAtlasFederatedSettingsOrganizationConfigs() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceMongoDBAtlasFederatedSettingsOrganizationConfigsRead,
+		Schema: map[string]*schema.Schema{
+			"federation_settings_id": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
doesn't this need org id?

Reply:
Added datasource with org_id parameter and tests